crates/gossip/src/gossip.rs

```diff
@@ -717,42 +652,6 @@ mod test {
             .await
             .unwrap();
 
-        harness_2
```

Comment:
Same here, I think this is a key part of the test. If it passes without it then something is wrong with the test